hack/rpm-deps.sh

```diff
@@ -20,6 +20,17 @@ virt_v2v="
   virtio-win
 "
 
+ovirt-imageio="
+  gcc
+  py3-pip
+  python3-dev
```

Comment:
```suggestion
  python3-devel
```

Reply:
how about using el9? then we can take it from https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/